devices.js

```diff
@@ -12632,6 +12632,26 @@ const devices = [
         },
         exposes: [e.temperature(), e.humidity(), e.battery()],
     },
+    {
+        zigbeeModel: ['898ca74409a740b28d5841661e72268d'],
+        model: 'ST30',
+        vendor: 'ORVIBO',
+        description: 'Temperature & humidity sensor',
+        fromZigbee: [fz.humidity, fz.temperature, fz.battery],
+        toZigbee: [],
+        meta: {configureKey: 2, battery: {voltageToPercentage: '3V_2500'}},
```

Comment:
`await reporting.batteryPercentageRemaining(endpoint2);` should already provide the `battery` (%) so not needed to convert voltage -> battery here (`battery: {voltageToPercentage: '3V_2500'}` can be removed I think)

Reply:
Yup!  Thanks!